mvapi.yaml

```
@@ -163,51 +203,39 @@ paths:
         '200':
           description: OK
         '404':
-            description: Judgment not found
+          $ref: '#/components/responses/JudgmentNotFound'
       security:
-        - {}
         - judgeAuth: []
-  '/invites/{pollId}':
+  '/judges/{judge-id}/':
```

```suggestion
  '/polls/{poll-id}/judges/{judge-id}/':
```

We do action on a judge *about a specific poll*.